### DISCUSSION TITLE
Format of book

### DISCUSSION THREAD

**Discussion by hungpham1001** [koreader/koreader#7484] [Ideas]
I read the same book on my kobo device, but there is the difference with koreader and stock reader of kobo as you see in the picture. If koreader can display the format as same as stock reader, it would be perfect
![PXL_20210402_032253082](https://user-images.githubusercontent.com/57803502/113377715-4d3f2880-939f-11eb-9dff-b137f3406276.jpg)
![PXL_20210402_032426248](https://user-images.githubusercontent.com/57803502/113377727-562ffa00-939f-11eb-9cf6-d0b2965ede7f.jpg)

**Comment by Frenzie** [comment]
That's probably `::first-letter`, which isn't supported yet.

**Comment by dosimple** [comment]
Are you sure, the displaying of kobo is perfect? In my view, the displaying of koreader feels better.
